shell/src/main/java/alluxio/cli/fsadmin/command/PathConfCommand.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.command;
+
+import alluxio.cli.Command;
+import alluxio.cli.CommandUtils;
+import alluxio.cli.fsadmin.pathconf.ListCommand;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.exception.AlluxioException;
+import alluxio.exception.status.InvalidArgumentException;
+import alluxio.util.CommonUtils;
+
+import org.apache.commons.cli.CommandLine;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Manages path level configuration.
+ */
+public final class PathConfCommand extends AbstractFsAdminCommand {
+  private enum SubCommand {
+    LIST("list", ListCommand.class),
+    ;
+
+    private final String mName;
+    private final Class<? extends Command> mClass;
```

Comment:
consider storing a `Map<String, BiFunction<Context, AlluxioConfiguration, ? extends Command>>` instead. Then we don't need reflection at all